docs/installation.rst

```diff
@@ -5,6 +5,26 @@ Installation
 .. note::
     This page documents how to install FAME on Ubuntu 20.04. FAME being written in Python, you can install it on the system of your choice.
 
+The easy way
+============
+
+Probably the most easy way to run fame is::
+
+    $ git clone https://github.com/certsocietegenerale/fame.git
+    $ cd fame/docker
+    $ cp fame.env.template fame.env
+    $ sed -i "s/#privileged: true/privileged: true/g" docker-compose.yml
```

Comment:
Could you explain why you the change from Docker out of Docker to Docker in Docker ? I do not have experience with any of them but the first one seemed good to me.

If we do chose to keep this Docker in Docker approach, we should explain that this step is optional and when it is necessary. Otherwise, we should directly uncomment it in the docker-compose file.

Reply:
In general, DooD is recommended in most cases : Unlike DinD, DooD does not require privileged containers, and this limit (a little bit) the possibilities for container evasion.
Note that container evasion is still possible with DooD (since we have access to the docker socket, we can spawn a privileged container on the host...) but It is considered cleaner to perform DooD anyway

Why did I switch to DinD, then?
Because of one of the limitations with DooD : [bind volumes between parent docker and child are very complex](http://ralphmcneal.com/dood-pitfalls-volume-mounts/), and dynamic volume creation is not supported (all volumes must be created before starting the parent docker)

Bind volumes are being used to transfer data between FAME modules and a fame worker instance. I could have reworked how fame modules transfers are being done (in fact, that's what https://github.com/certsocietegenerale/fame_modules/pull/27 tried to do, if I'm not mistaken) but I instead opted out to switch to DinD

About `sed` specifically:
If you don't execute the sed command, then modules using docker (url_preview, etc) won't work. Modules which are not using docker will work just fine
I made it this way to "force" users to understand the container is privileged. 

Do you confirm that I should uncomment it directly in the docker-compose?

Reply:
I removed the `sed` part in the doc. I'm assuming this can be marked as resolved